src/js/api/urls.js

```diff
@@ -17,6 +17,7 @@ export const STOCK_MOVEMENT_BY_ID = (id) => `${STOCK_MOVEMENT_API}/${id}`;
 export const STOCK_MOVEMENT_PENDING_SHIPMENT_ITEMS = `${STOCK_MOVEMENT_API}/pendingRequisitionItems`;
 export const STOCK_MOVEMENT_INCOMING_ITEMS = `${STOCK_MOVEMENT_API}/shippedItems`;
 export const STOCK_MOVEMENT_UPDATE_STATUS = (id) => `${STOCK_MOVEMENT_API}/${id}/status`;
+export const STOCK_MOVEMENT_UPDATE = (id) => `${STOCK_MOVEMENT_API}/${id}/updateRequisition`;
```

Comment:
i would call it UPDATE_REQUISITION, we have also UPDATE_STATUS, UPDATE_ITEMS, etc. so I feel like something is missing here